app/components/results-list/component.js

```diff
@@ -50,11 +50,11 @@ export default Ember.Component.extend({
             let queryParams = {};
             let page = Number(this.parameters["page"]);
             if (!page) {
-                page = 0;
+                page = 1;
```

Comment:
You can do `let page = Number(this.parameters.page) || 1;` to get rid of this if statement (same thing below)